### PR TITLE
[feature](hive/iceberg)add doris's version in table properties

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

issue #31442
when create a external table, we can add doris's version in table's properties.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java [hunk 1]
```diff
@@ -86,6 +87,8 @@ public abstract class ExternalCatalog
     private static final Logger LOG = LogManager.getLogger(ExternalCatalog.class);
 
     public static final String ENABLE_AUTO_ANALYZE = "enable.auto.analyze";
+    public static final String DORIS_VERSION = "doris_version";
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java:90]:
```suggestion
    public static final String DORIS_VERSION = "doris.version";
```

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Review by wsjz — APPROVED** [review]
LGTM

**Pull request merged**